pallets/did/src/lib.rs

```diff
@@ -537,12 +588,6 @@ where
 			);
 		};
 
-		// Verify that the operation counter is greater than the stored one
```

Comment:
Could you explain the why this does not hold true anymore? Sorry if I has been explained already.

Reply:
Yeah we basically briefly discussed with @weichweich that if we allow users to put ANY value greater than the current one, we might face the situation in which we reach the maximum counter value. And as we do not have at the moment a strategy to wrap those around, we are just making it harder to happen by forcing users to only nonces that are 1 greater than the last one used.